From 5e161f1907cd3b91a142d64bf1af2c22108b98e3 Mon Sep 17 00:00:00 2001
From: Lukas Nykryn <lnykryn@redhat.com>
Date: Fri, 21 Sep 2012 12:33:32 +0200
Subject: [PATCH] login: missing break for getopt ARG_NO_ASK_PASSWORD in
 loginctl (cherry picked from commit
 5d5e98eb8c859d5a85fe6cd5e3cc433bd8a096ba)

---
 src/login/loginctl.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/login/loginctl.c b/src/login/loginctl.c
index 9a43685..1d548ed 100644
--- a/src/login/loginctl.c
+++ b/src/login/loginctl.c
@@ -1768,6 +1768,7 @@ static int parse_argv(int argc, char *argv[]) {
 
                 case ARG_NO_ASK_PASSWORD:
                         arg_ask_password = false;
+                        break;
 
                 case ARG_KILL_WHO:
                         arg_kill_who = optarg;
